model_analyzer/config/input/config_command.py

```diff
@@ -95,40 +96,137 @@ def set_config_values(self, args):
         """
 
         # Config file has been specified
+        yaml_config = self._load_yaml_config(args)
+        self._check_for_illegal_config_settings(args, yaml_config)
+        self._set_field_values(args, yaml_config)
+        self._preprocess_and_verify_arguments()
+        self._autofill_values()
+
+    def _load_yaml_config(self, args: Namespace) -> Dict[str, List]:
```

Comment:
Does this pass type checking? We can return None

Reply:
It did pass type checking, but yes, it should be Optional. I've made the change.